west.yml

```diff
@@ -69,7 +69,6 @@ manifest:
     - name: hal_espressif
       revision: bd2e2d8dcdc19defdb1bf6e07fb94f62bc99c925
       path: modules/hal/espressif
-      west-commands: west/west-commands.yml
```

Comment:
For the record, I am not in favour of the option proposed,  but just in case this is what the TSC decides for, I am against making this change to the manifest and enforcing this instantly at the same time it becomes a policy and part of the documentation. There need to be some grace period for the owners of such modules to adapt to the new policy.

Furthermore, the west commands  in this module do more than just pull blobs, so the above change will break building and flashing this platform in general.

Reply:
Shouldn't the change request be that the west command extension in the HAL be modified to remove in-tree support for pulling binaries? 

I don't know what all the espressif extension does but their python script indicates "... enable proper submodules update, toolchain installation and serial port monitor logging". I suspect the last two are things that are not counter to this particular PR if we decide to go in this direction.

Reply:
Agreed, dropped the manifest change.